source/slang/slang-ir-lower-result-type.h

```diff
@@ -6,9 +6,10 @@
 namespace Slang
 {
 struct IRModule;
+class TargetProgram;
 class DiagnosticSink;
 
 /// Lower `IRResultType<T,E>` types to ordinary `struct`s.
-void lowerResultType(IRModule* module, DiagnosticSink* sink);
+void lowerResultType(IRModule* module, TargetProgram* targetProgram, DiagnosticSink* sink);
```

Comment:
The target program now needs to be given to passes that want to determine the sizes of types, as they can be target-dependent. A lot of passes have this same change, so I'm only commenting about it here.